test_data/provider_defs/open_weather.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "EdrDataProviderDefinition",
+  "id": "0668d980-6c82-47c4-b1d9-1096f6770666",
+  "name": "Open Weather",
```

Comment:
Could you tell me a path to a layer that works?

Reply:
collections!GFS_isobaric!temperature!1000

Reply:
![grafik](https://github.com/geo-engine/geoengine/assets/4947061/6b9c665a-6087-45fb-90b7-dac538b4690a)

I still don't see anything

Reply:
You need to expand a layer on the right hand panel and look at the available time range. Data from OpenWeather is only provided starting from the current time and a few days after that.
Also, you need to keep in mind that because of the bug I mentioned tiles will only be displayed if you are zoomed out.